Documentation/building/windows-instructions.md

```diff
@@ -76,6 +76,23 @@ For more details, or to test an individual project, see the [developer guide top
 4. Set breakpoint appropriately
 5. F5 (Debug)
 
+### Debugging NETFX tests in Visual Studio
+
+Once you've build the source code for netfx from the root (`build.cmd -framework:netfx`) follow this steps:
+
+1. Build the test project through `msbuild /p:targetgroup=netfx`
+2. Open Visual Studio
+3. Open xunit runner for your test project `<corefxpath>\bin\<OSGroup.Architecture.BuildConfiguration>\<YOURPROJECT>\<TARGETGROUP>\xunit.console.exe`
+    * Example: `D:\corefx\bin\AnyOS.AnyCPU.Debug\System.Runtime.Extensions.Tests\netstandard\xunit.console.exe`
+4. Open project properties and fill in the next information:
+    * Arguments -> ```<YOURPROJECT>.dll -noshadow -xml testResults.xml -notrait Benchmark=true -notrait category=nonnetfxtests  -notrait category=OuterLoop -notrait category=failing -notrait category=nonwindowstests```
+         * Note that this arguments change if you are running/debugging Outerloop tests (You would need to delete `-notrait category=Outerloop`. A good way to get them right is to build your test project and copy xunit's arguments from RunTests.cmd inside the test project bin folder.
+    * Debugger Type -> Managed (v4.6, v4.5, v4.0)
+    * Environment -> you need to add an environment variable as follows:
+         * DEVPATH -> `<corefxpath>\bin\testhost\netfx-Windows_NT-Debug-x64\`
+5. Set breakpoint appropriately
+6. F5 (Debug)
```

Comment:
I just tried this and it works perfect. Didn't add this because since this didn't work before and now I remember it was fixed but I didn't tried it ever since. Thanks for pointing it out. Updating the PR.